app/controllers/controllershelpers/HomeCardGenerator.scala

```diff
@@ -104,7 +104,7 @@ class HomeCardGenerator @Inject() (
     val isItsaEnrolled = enrolmentsHelper.itsaEnrolmentStatus(request.enrolments).isDefined
     val saView         = saAndItsaMergeView((current.currentYear + 1).toString, isItsaEnrolled)
 
-    request.trustedHelper.flatMap(_ => None).orElse {
+    request.trustedHelper.map(_ => None).getOrElse {
       if (isItsaEnrolled || request.isSa) {
         Some(saView)
```

Comment:
tbh I think this would be clearer if you just say request.trustedHelper match {case Some(_) => None case _ => etc... }

Reply:
Or you could match like this: (request.trustedHelper, isItsaEnrolled || request.isSa, configDecorator.pegaSaRegistrationEnabled) match